astrosat/utils/utils_logging.py

```diff
@@ -53,10 +52,21 @@ def emit(self, record):
                 ],
             )
 
-            db_record = DatabaseLogRecord.objects.create(
-                logger_name=record.name,
-                level=record.levelno,
-                message=record.getMessage(),
-                trace=trace,
-            )
+            try:
+                uuid = getattr(record, 'uuid')
+
+                db_record = DatabaseLogRecord.objects.create(
+                    logger_name=record.name,
+                    level=record.levelno,
+                    message=record.getMessage(),
+                    uuid=uuid,
```

Comment:
I suspect (though I'm not certain) that b/c you are always passing `uuid` here even if it's None, that this will have the effect of sometimes creating a `DatabaseLogRecord` w/ a blank `uuid` instead of an automatically-generated one.  To get around this you could have a big if statement:


```
  uiid = getattr(record, "uuid", None)
  if uuid:
    db_record = DatabaseLogRecord.objects.create(
       logger_name=record.name,
       level=record.levelno,
       message=record.getMessage(),
       trace=trace,
      uuid=uuid,
    )
else:
    db_record = DatabaseLogRecord.objects.create(
       logger_name=record.name,
       level=record.levelno,
       message=record.getMessage(),
       trace=trace,
)
```

This is a bit cumbersome, so you could also do something like this:

```
uuid = getattr(record, "uuid", uuid.uuid4())
```

Or you could build up the kwargs to the create method like this:
```
uuid = getattr(record, "uuid", None)
db_record_kwargs = {
  "logger_name": record.name,
   "level": record.levelno,
   "message": record.getMessage(),
    "trace": trace,
}
if uuid:
   db_record_kwargs["uuid"] = uuid
db_record = DatabaseLogRecord.objects.create(**db_record_kwargs)
```

These are all equally valid (and equally untested), just pick your favourite.